projects/elife.yaml

```diff
@@ -926,13 +926,17 @@ journal-cms:
         - https://github.com/elifesciences/builder-base-formula
         - https://github.com/elifesciences/api-dummy-formula
     aws:
+        # deprecated
         ec2:
-            ami: ami-92002785 # Ubuntu 14.04, deprecated
+            ami: ami-14a1d06e # GENERATED created from basebox--1604
+                              # us-east-1, build date 20170811, hvm:ebs-ssd, AMI built on 20171215
         ports:
             - 22
             - 443
             - 80:
                 cidr-ip: 10.0.0.0/16 # access via VPC ip range only
+        ext:
```

Comment:
this will also affect formula builds, but it's probably for the best as they may run out of space otherwise. The additional times it takes to provision shouldn't be a big deal, it's RDS that takes tens of minutes.

Reply:
I agree. I'll revert those `ext` changes. `journal-cms` is just so fat every case is a special case. I will add a special alt-config intended for adhoc instances though with rds and the large ext disk.